basic_nodes/src/main/java/club/mondaylunch/gatos/basicnodes/BasicNodes.java

```diff
@@ -46,6 +46,8 @@ public static void init() {
         .register("truthiness", new TruthinessNodeType());
     public static final NegationNodeType NEGATION = NodeType.REGISTRY
         .register("negation", new NegationNodeType());
+    public static final ParseNumberFromStringNodeType PARSE_STRING_FROM_NUMBER = NodeType.REGISTRY
+        .register("parse_string_from_number", new ParseNumberFromStringNodeType());
```

Comment:
This is inconsistent (both 'parse number from string' and 'parse string from number').

Personally, I'd go with 'parse string to number' since it's the string that's being parsed, and we get a number

Reply:
Genuine typo, my bad